<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="font">
       <font>
        <family>仿宋</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionnew"/>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
    <addaction name="actionsaveTo"/>
    <addaction name="separator"/>
    <addaction name="actionsetting"/>
    <addaction name="separator"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionundo"/>
    <addaction name="separator"/>
    <addaction name="actioncut"/>
    <addaction name="actioncopy"/>
    <addaction name="actionpaste"/>
    <addaction name="actiondelete"/>
    <addaction name="separator"/>
    <addaction name="actionall"/>
    <addaction name="actiondatetime"/>
    <addaction name="separator"/>
    <addaction name="actionfont"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>查看</string>
    </property>
    <widget class="QMenu" name="menucoding">
     <property name="title">
      <string>编码方式</string>
     </property>
     <addaction name="actionUTF_8"/>
     <addaction name="actionGBK"/>
     <addaction name="actionISO_8859_1"/>
     <addaction name="actionUTF_16_LE"/>
     <addaction name="actionUTF_16_BE"/>
     <addaction name="actionGB2312"/>
    </widget>
    <addaction name="actionsearch"/>
    <addaction name="actionreplace"/>
    <addaction name="actiongoto"/>
    <addaction name="separator"/>
    <addaction name="actionstatusBar"/>
    <addaction name="actionwrap"/>
    <addaction name="separator"/>
    <addaction name="menucoding"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="viewMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionnew">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionsaveTo">
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionsetting">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actioncut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actiondelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionall">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actiondatetime">
   <property name="text">
    <string>当前时间</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionfont">
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="actionsearch">
   <property name="text">
    <string>搜索</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionreplace">
   <property name="text">
    <string>替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actiongoto">
   <property name="text">
    <string>转到</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionstatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>状态栏</string>
   </property>
  </action>
  <action name="actionwrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>自动换行</string>
   </property>
  </action>
  <action name="actionUTF_8">
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="actionGBK">
   <property name="text">
    <string>GBK</string>
   </property>
  </action>
  <action name="actionISO_8859_1">
   <property name="text">
    <string>ISO-8859-1</string>
   </property>
  </action>
  <action name="actionUTF_16_LE">
   <property name="text">
    <string>UTF-16 LE</string>
   </property>
  </action>
  <action name="actionUTF_16_BE">
   <property name="text">
    <string>UTF-16 BE</string>
   </property>
  </action>
  <action name="actionGB2312">
   <property name="text">
    <string>GB2312</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
